### DISCUSSION TITLE
install package not from pypi

### DISCUSSION THREAD

**Discussion by Jzhenli** [beeware/briefcase#2070] [Q&A] [answered]
Hi, How to install python package from piwheels (not from pypi), I try as follows
![image](https://github.com/user-attachments/assets/81f1e3cf-13ea-4128-aaaa-bff52f4b7149)

but it will rasie error
![image](https://github.com/user-attachments/assets/56125cb6-1bc3-41b2-acf1-c761e47b7102)

would u give some suggestion? Thanks.

**Comment by rmartin16** [comment, marked as answer]
Try using an equal sign, i.e. `=`, between `-i` and the URL.

More info: https://github.com/beeware/briefcase/issues/1270#issuecomment-1656984001
